core-api/src/main/java/com/sequenceiq/distrox/api/v1/distrox/model/DistroXScaleV1Request.java

```diff
@@ -27,6 +28,10 @@
     @ApiModelProperty(value = InstanceGroupNetworkScaleModelDescription.NETWORK_SCALE_REQUEST)
     private NetworkScaleV1Request networkScaleRequest;
 
+    private AdjustmentType adjustmentType = AdjustmentType.EXACT;
+
+    private Long threshold;
+
```

Comment:
Is the missing @ApiModelProperty intentional?

Reply:
hmm.. I will add it, I think I missed it